cumulus/primitives/utility/src/lib.rs

```diff
@@ -526,6 +529,8 @@ impl<
 mod test_xcm_router {
 	use super::*;
 	use cumulus_primitives_core::UpwardMessage;
+	use frame_support::assert_ok;
```

Comment:
You can look into srlabs finding [#336](https://github.com/paritytech-secops/srlabs_findings/issues/336).  
It is to prevent the sending of messages that will surely not decode, because of nesting limits.